### PR TITLE
[fix bug 1498290] Expose locale versions of Content Blocking Tour

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Exposes content blocking tour variations to all locales.
- Adds some last minute string changes due to FastBlock not shipping in FF 63.
- Needs to be in production before FF 63 launch (10/23).

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1498290
https://bugzilla.mozilla.org/show_bug.cgi?id=1499589

### CODE REVIEW

**Comment by alexgibson** [comment]
@peiying2 are we OK to remove this restriction, assuming enough locales now have the new strings?

**Comment by peiying2** [comment]
@alexgibson Thanks for checking.  According to this page, we have reached beyond the threshold of 80% users with the updated content: https://l10n.mozilla-community.org/langchecker/?locale=all&website=0&file=firefox/tracking-protection-tour.lang. We can remove the restrictions.

**Comment by alexgibson** [comment]
@peiying2 there have been some last minute string changes due to FastBlock not shipping in Firefox 63. Thankfully only 2 strings, which I have added behind the l10n tag `remove_fastblock_1499589`. Let me know if this works for you, thanks.

**Comment by flodolo** [comment]
It works, thanks (covering during PTO)

File: bedrock/firefox/templates/firefox/tracking-protection-tour/variation-1.html [hunk 1]
```diff
@@ -32,7 +38,11 @@ <h1>{{ _('Content Blocking') }}</h1>
 
 {% block panel_2 %}
   <h2>{{ _('Differences to expect') }}</h2>
+  {% if l10n_has_tag('remove_fastblock_1499589') %}
+  <p>{{ _('Content blocking keep parts of pages or entire pages from loading.') }}</p>
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/tracking-protection-tour/variation-1.html]: Completely missed this :-\

A localizer noticed that "keep" should "keeps", so I checked the copy, and this should be "can keep".

File: bedrock/firefox/templates/firefox/tracking-protection-tour/variation-2.html [hunk 2]
```diff
@@ -32,7 +32,11 @@ <h1>{{ _('Content Blocking') }}</h1>
 
 {% block panel_2 %}
   <h2>{{ _('Differences to expect') }}</h2>
-  <p>{{ _('Content blocking can help pages load faster, but it can also  keep parts of pages or entire pages from loading.') }}</p>
+  {% if l10n_has_tag('remove_fastblock_1499589') %}
+  <p>{{ _('Content blocking keep parts of pages or entire pages from loading.') }}</p>
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/tracking-protection-tour/variation-2.html]: Same here

File: bedrock/firefox/templates/firefox/tracking-protection-tour/variation-1.html (hunk 1 above)
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/tracking-protection-tour/variation-1.html]: Bah, sorry this was my mistake, will fix asap

**Comment by alexgibson** [comment]
@flodolo updated, sorry for missing that the first time 😞

**Comment by alexgibson** [comment]
Still waiting on confirmation in the bug for the go-ahead, but we need to get this to production this week - so it would be good to get this code reviewed so we're all ready.

**Comment by alexgibson** [comment]
Ok, we have [go-ahead](https://bugzilla.mozilla.org/show_bug.cgi?id=1499589#c4) to merge this once reviewed

**Comment by alexgibson** [comment]
Notes for testing:


1. Test using Firefox Beta
2. Set browser.contentblocking.ui.enabled to true
3. Enable UITour for demo and/or local dev.
4. Open about:preferences#privacy and make sure content blocking is enabled.

Specific URLs to test include:

- http://localhost:8000/en-US/firefox/tracking-protection/start/?variation=1&step=1
- http://localhost:8000/en-US/firefox/tracking-protection/start/?variation=2&step=1

**Pull request merged**